.gitignore

```diff
@@ -15,3 +15,9 @@ bin/**
 
 # Don't checkin the binary
 grpc-go-redact
+
```

Comment:
just delete the files from the repo

Reply:
actually youre trying to regenerate them in the tests now? we shouldnt have make commands that reference files made within tests?

Reply:
These files are being generated when you run `make test`. Previsouly, if you changed the String func implementation, you had to manually regenerate test files and it was not clear how to do that. Now you just run `make test` and the Makefile will hadle everything for you.